ebmlite/ideParsingExample.py

```diff
@@ -26,71 +24,71 @@ def getTypeMatch(el, elType):
 chList = getTypeMatch(recProps, chListType).dump()
 
 # Print the ID and name of each channel.
-chIdNames = [[ch['ChannelID'], ch['ChannelName']] for ch in chList['Channel']]
-print('Channels:\r\n%s\r\n' % (str(chIdNames)))
+chIdNames = [[ch[b'ChannelID'], ch[b'ChannelName']] for ch in chList[b'Channel']]
+print(b'Channels:\r\n%s\r\n' % (str(chIdNames)))
 
 # Define the channel that we'll be working with.
 chId = 8
 
 # Get the channel that we want to work with from the list of channels.
-chEl = next(ch for ch in chList['Channel'] if ch['ChannelID'] == chId)
+chEl = next(ch for ch in chList[b'Channel'] if ch[b'ChannelID'] == chId)
 
 # Print the ID and name of each subchannel.
-schIdNames = [[sch['SubChannelID'], sch['SubChannelName']] for sch in chEl['SubChannel']]
-print('Subchannels:\r\n%s\r\n' % (str(schIdNames),))
+schIdNames = [[sch[b'SubChannelID'], sch[b'SubChannelName']] for sch in chEl[b'SubChannel']]
+print(b'Subchannels:\r\n%s\r\n' % (str(schIdNames),))
 
 # Define the subchannel that we'll be working with.
 schId = 0
 
 # Get the channel that we want to work with from the list of channels.
-schEl = next(sch for sch in chEl['SubChannel'] if sch['SubChannelID'] == schId)
+schEl = next(sch for sch in chEl[b'SubChannel'] if sch[b'SubChannelID'] == schId)
 
 # Collect all the channelDataBlocks into a list.
 chDataType = schema[0xA1]
-dataBlocks = filter(lambda x: type(x) == chDataType, ideRoot)
+dataBlocks = [x for x in ideRoot if type(x) == chDataType]
 
 # Filter the dataBlocks to only include blocks for the channel we want.
 chIdType = schema[0xB0]
 dataBlocks = [block for block in dataBlocks if getTypeMatch(block, chIdType).value == chId]
 
 # Get the raw data from each ChannelDataBlock, and convert to an array.
-rawData = b''
+rawData = ''
 payloadType = schema[0xB2]
 for block in dataBlocks:
-    rawData += block.dump()['ChannelDataPayload']
-rawData = np.fromstring(str(rawData), dtype=chEl['ChannelFormat'][1])
-rawData.resize((len(rawData)//3, 3))
+    rawData += block.dump()[b'ChannelDataPayload']
+rawData = np.fromstring(str(rawData), dtype=chEl[b'ChannelFormat'][1])
+rawData.resize((old_div(len(rawData),3), 3))
```

Comment:
Can this be done with a regular Python expression? I don't see `old_div()` being imported or defined.